docs/continuing_runs.md

```diff
@@ -0,0 +1,204 @@
+# Continuing and editing runs
+
+The modularity of HADDOCK3 allows users to take parts of a run and use them as
+starting points for new runs using different configuration files. There are
+several ways users can continue or expand previous runs. The index below lists
+those possibilities:
+
+1. Restart a previous run from a given step
+1. Expanding a previous run from a given step
+1. Editing and expanding a previous run from a given step
+1. Copy a run to a new folder
+1. Extend an unknown run
+1. Use a module as a seed for a new run
```

Comment:
Kind of related to copy a run to a new folder - i.e. copy partially a run to a new folder

Reply:
Yes. I had them separate to avoid overloading each example. So that all the possibilities become clear and are explained in separate steps. But we can change if in the end we see it's best explained differently